### DISCUSSION TITLE
Add sync correction for group speaker?

### DISCUSSION THREAD

**Discussion by kizovinh** [music-assistant/support#2549] [Q&A] [answered]
Is it possible to add sync correction between different speaker from different vendor? I am using homepod with google nest mini, and the nest mini's audio is about 1s slower than the homepod. So if I can add 1s delay to the nest, they would be nice.

**Comment by marcelveldt** [comment, marked as answer]
No that is not possible

**Reply by Jeremywp123** [reply]
Please make it possible, I just need to be able to add about a 500ms delay to my Nest hubs when they are cast to in conjunction with my Google TV, even if it's not perfect it would be a ton better than a half second delay between the speakers.

**Reply by OzGav** [reply]
Group them with the google home app and they should play in sync

**Comment by Jeremywp123** [comment]
But then you don't have the ability to spin send to change devices on the fly.Turns out they added this feature.Settings/players/(device) and enable advanced options. Under the sendspin drop down there is a delay slider.Sent from my Galaxy

**Reply by OzGav** [reply]
This may be moot soon as Google is updating the cast firmware which will permanently break the sendspin bridge
